src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,9 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+
+        if ((a <= b && b <= c) || (a >= b && b >= c)) return true;
+        else return false;
```

Comment:
You don't need `if-else` statement here as the result of your expression is `boolean` - so just return your expression.